src/components/customComponents/EmptyNote.js

```diff
@@ -0,0 +1,51 @@
+import { Card, CardContent, CardActions, Typography, IconButton, TextField } from "@mui/material";
+import PushPinOutlinedIcon from '@mui/icons-material/PushPinOutlined';
+import PushPinIcon from '@mui/icons-material/PushPin';
+import LabelOutlinedIcon from '@mui/icons-material/LabelOutlined';
+import DeleteOutlinedIcon from '@mui/icons-material/DeleteOutlined';
+import ArchiveOutlinedIcon from '@mui/icons-material/ArchiveOutlined';
+import ColorLensOutlinedIcon from '@mui/icons-material/ColorLensOutlined';
+import { NotelyTheme } from '../../styles';
+import { useNote } from '../../contexts/note-context';
+import { ColorPallete } from "./ColorPallete";
+import { AddTag } from "./AddTag";
```

Comment:
Instead of writing relative paths involving multiple ../../contexts/note-context.
You can start using absolute paths in your projects which is very easy to set up and if used can make your code imports look readable . Here is a link related to the same 
https://stackoverflow.com/questions/63067555/how-to-make-an-import-shortcut-alias-in-create-react-app

Reply:
Thanks for pointing this out. I tried but couldn't make it work. I will try again after the feature dev is completed.Will reach out if needed.